uber/site_sections/accounts.py

```diff
@@ -177,10 +177,21 @@ def login(self, session, message='', original_location=None, **params):
     def homepage(self, session, message=''):
         if not cherrypy.session.get('account_id'):
             raise HTTPRedirect('login?message={}', 'You are not logged in')
-        attendees = session.viewable_attendees().limit(c.ROW_LOAD_LIMIT)
+        
         return {
-            'attendees': attendees.all(),
-            'message': message
+            'message': message,
+            'site_sections': [key for key in session.access_query_matrix().keys() if getattr(c, 'HAS_' + key.upper() + '_ACCESS')]
+            }
+        
+    @public
+    def attendees(self, session, query=''):
+        if not cherrypy.session.get('account_id'):
+            raise HTTPRedirect('login?message={}', 'You are not logged in')
+        
+        attendees = session.access_query_matrix()[query].limit(c.ROW_LOAD_LIMIT).all() if query else None
```

Comment:
It's good that we're no longer loading anything by default - that probably addresses this.

Here's a sanity check question about ``c.ROW_LOAD_LIMIT``: should we lower it to be a much lower number?  There are basically two types of people who will be using this page:

1) Admins with permission to see everything, who should be (and will be!) directed to use a different form to see all entries.

2) People who ONLY have access to this page and who by definition only need to see a relatively low number of people, e.g. other people in their department.

For people in (1) we could set this to be really really low, like 50, because it won't matter if stuff is cut off.

For people in (2) we want the number to be high enough that it's going to load everyone in their departments or whatever.  However, 2000 is probably on the high side for this.

Could we get away with reducing this to something like 500?  I don't know of any use cases where that wouldn't be enough, though perhaps it would if e.g. panel admins need to be able to see all panelists or something.

Reply:
For this page to truly be useful, it should ideally get all the necessary attendees... but for now, yea, let's just limit it to 500. The real solution to all this is to implement serverside paging, at any rate.